src/components/TodoInfo/TodoInfo.tsx

```diff
@@ -1 +1,20 @@
-export const TodoInfo = () => {};
+import classNames from 'classnames';
+import { Todo } from '../../types';
+import { UserInfo } from '../UserInfo';
+
+type Props = {
+  todo: Todo,
+};
+
+export const TodoInfo: React.FC<Props> = ({ todo }) => {
+  return (
+    // eslint-disable-next-line
+    <article data-id={todo.id} className={classNames('TodoInfo', {'TodoInfo--completed': todo.completed})}>
+      <h2 className="TodoInfo__title">
+        {todo.title}
+      </h2>
+
+      <UserInfo user={todo.user} />
```

Comment:
[Checklist](https://github.com/mate-academy/react_add-todo-form/blob/master/checklist.md):

[REACT KNOWLEDGE] - Don't render the component if the property that you pass to the component has null or undefined value.